lib/docs.js

```diff
@@ -0,0 +1,19 @@
+// Create object for providing easy access to documentation
+
+function Docs() {
+  var fs = require('fs'),
+      path = require('path'),
+      rules = fs.readdirSync(path.resolve('node_modules/eslint/docs/rules')),
```

Comment:
Can we determine & resolve this path relative to [`__dirname`](https://nodejs.org/docs/latest/api/globals.html#globals_dirname)? I think this would be less fragile (and easier to test).

Also, I know I'm a broken record, but can we write tests? :neckbeard: